test/unit/controllers/adhoc_payment/post_index_controller_test.js

```diff
@@ -38,7 +38,7 @@ describe('adhoc payment submit-amount controller', function () {
         supertest(createAppWithSession(getApp()))
           .post(paths.pay.product.replace(':productExternalId', product.external_id))
           .send({
-            'payment-amount': '125.50',
+            'payment-amount': '9.95',
```

Comment:
could `payment-amount` ever be e.g. `5.3` or `5`?

Reply:
Actually, it turns out we have validation a few lines above that filters out rejects anything that doesn’t match either a whole number (like `125`) a whole number followed by a dot followed by two periods (like `125.50`). So `5` will get treated as I described above but `5.3` will fail the validation.